backend/dal/Helpers/Extensions/ProjectExtensions.cs

```diff
@@ -57,7 +56,7 @@ public static IQueryable<Entity.Project> GenerateQuery(this PimsContext context,
             if (!String.IsNullOrWhiteSpace(filter.ProjectNumber))
                 query = query.Where(p => EF.Functions.Like(p.ProjectNumber, $"%{filter.ProjectNumber}%"));
             if (!String.IsNullOrWhiteSpace(filter.Name))
-                query = query.Where(p => EF.Functions.Like(p.Name, $"%{filter.Name}%"));
+                query = query.Where(p => EF.Functions.Like(p.Name, $"%{filter.Name}%") || EF.Functions.Like(p.ProjectNumber, $"%{filter.Name}%"));
```

Comment:
is this safe? dumb example but if I name my project "2" do I expect to get back all of the projects with 2 in the project number?

Reply:
I'm not entirely certain what's best here. We don't current provide a way to find a project by its `ProjectNumber` in the UI.  Which is one of the most common ways to find a project.  So I did this to essentially bypass the need to add yet another filter on the project list view page.   Essentially a workaround to get us closer to user friendly.